src/ray/util/shared_lru.h

```diff
@@ -100,8 +107,10 @@ class SharedLruCache final {
     return true;
   }
 
-  // Look up the entry with key `key`. Return nullptr if key doesn't exist.
-  std::shared_ptr<Val> Get(const Key &key) {
+  // Look up the entry with key `key`. Return std::nullopt if key doesn't exist.
+  // If found, return a copy for the value.
+  template <typename KeyLike>
+  std::shared_ptr<Val> Get(KeyLike &&key) {
```

Comment:
ditto, why need &&? I think const& is fine

Reply:
Answered above.